stripe_webhook/src/webhook.rs

```diff
@@ -138,22 +137,24 @@ struct Signature<'r> {
 
 impl<'r> Signature<'r> {
     fn parse(raw: &'r str) -> Result<Signature<'r>, WebhookError> {
-        let headers: HashMap<&str, &str> = raw
```

Comment:
Would be a strange attack, but seems good to avoid collecting every pair in the signature since it could be malicious